apps/bbsync/tests/test_query.py

```diff
@@ -213,6 +215,47 @@ def test_generate_component(self, workflow_state, result):
         bbq = FlawBugzillaQueryBuilder(flaw)
         assert bbq.query["component"] == result
 
+    @pytest.mark.parametrize(
+        "workflow_state,result",
+        # Same as for test_generate_component_before_bz_sync.
+        # Explicit is better than implicit.
+        TEST_GENERATE_COMPONENT_WITH_DRAFT,
```

Comment:
What is meant by this line? Every unit test should be explicit, but looking at other tests, they do not seem to be (e.g. constants and list comprehension in `parametrize`).

Reply:
I failed in conveying the meaning that instead of parametrizing whether it has been synced to bugzilla yet or instead of running only a few selected combinations, there's basically a copy&paste of the parametrize parameters.

Is it acceptable now?

Reply:
If you mean copy&paste of `parametrize` in `test_generate_component_before_bz_sync` and `test_generate_component_after_bz_sync_with_draft_component`, you can write it in one test this way:
```
    @pytest.mark.parametrize(
        "workflow_state,result",
        [
            (WorkflowModel.WorkflowState.NOVALUE, "vulnerability"),
            (WorkflowModel.WorkflowState.NEW, "vulnerability-draft"),
            (WorkflowModel.WorkflowState.TRIAGE, "vulnerability"),
            (WorkflowModel.WorkflowState.PRE_SECONDARY_ASSESSMENT, "vulnerability"),
            (WorkflowModel.WorkflowState.SECONDARY_ASSESSMENT, "vulnerability"),
            (WorkflowModel.WorkflowState.DONE, "vulnerability"),
            (WorkflowModel.WorkflowState.REJECTED, "vulnerability-draft"),
        ],
    )
    @pytest.mark.parametrize("has_meta_attr", [True, False])
    def test(self, workflow_state, result, has_meta_attr):
        """
        ...
        """
        flaw = FlawFactory(workflow_state=workflow_state, meta_attr={})
        if has_meta_attr:
            flaw.meta_attr["bz_component"] = "vulnerability-draft"

        bbq = FlawBugzillaQueryBuilder(flaw)
        assert bbq.query["component"] == result
```

I think this is still clear as both empty `meta_attr` and `vulnerability-draft` in `meta_attr` have the same `result`.

Reply:
But the current state is also good.

Reply:
Thanks, fixed mostly using your suggestion.